server/api/crud/files.py

```diff
@@ -48,6 +48,13 @@ def delete_artifact_data(
         secrets: dict = None,
     ):
         secrets = secrets or {}
+
+        # Verify that, similar project secrets, secrets passed by users are not internal.
```

Comment:
may you update the comment? it is unclear why it is not allowed

Reply:
@theSaarco why are we not permitting internal secrets when deleting artifact data?

Reply:
Internal secrets are, as their name implies, internal for the use of MLRun only. If the user can pass here any arbitrary secret to override its value, then we allow the user to interfere with how MLRun manages its secrets internally and can cause unexpected results.
Even if at this flow all we do is go to the storage and delete a file, we should still block that option.